### DISCUSSION TITLE
Is there any way to auto reload the web page on reload in fastapi ?

### DISCUSSION THREAD

**Discussion by AyushSehrawat** [fastapi/fastapi#6197] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, Request
from fastapi.staticfiles import StaticFiles
from fastapi.templating import Jinja2Templates

app = FastAPI()
app.mount("/static", StaticFiles(directory="./templates/static"), name="static")
templates = Jinja2Templates(directory="./templates")

@app.get("/")
def index(request : Request):
    return templates.TemplateResponse("index.html", {"request": request})

if __name__ == "__main__":
    import uvicorn
    uvicorn.run(
        "main:app",
        host="127.0.0.1",
        port=8000,
        reload_includes=["*.html", "*.css", "*.js"],
        reload_excludes=["./node_modules"],
        reload_dirs=["./templates", "./templates/static"],
        reload=True,
    )
```

### Description

I have added reload for html css js, and i want to reload the browser too on reload of FastAPI..Is there any way ? Currently the app only reloads on editing html css js files and py files...
Source Code : https://github.com/AyushSehrawat/secret-space

Is there any way to reload browser with reload of app ? Similar issue https://github.com/tiangolo/fastapi/issues/1981 can be found here, but some of them didn't work and i wanted to know if there is some better way ? 

### Operating System

Linux, Windows

### FastAPI Version

0.87.0

### Python Version

Python 3.10.4

**Comment by yinziyan1206** [comment]
FastAPI is a kind of web framework for backend application. You should use frontend framework to monitor the project version and reload the browser.  Or you can create a router in FastAPI to hash the html/js code, then get the request per second on the website. Just reload the browser when it is different.

**Comment by Kludex** [comment, marked as answer]
https://github.com/florimondmanca/arel

**Comment by AyushSehrawat** [comment]
> https://github.com/florimondmanca/arel

I guess i will go with this thanks
